tutorials/applications/Shadow estimation for VQS.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>I have a display pbm here.</p><p> </p><p>Also, I do not find the sentence "[...], meaning that on each qubit that o acts non-trivially upon, it must act with the same Pauli matrix as p" clear enough. Reformulate ?</p><p> </p><p>"[...] for each qubit that o acts non-trivially upon, the action is the same Pauli in o and p" or something like that (not convinced by my own proposition).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Great</p>